apps/grader-host/package.json

```diff
@@ -39,17 +39,17 @@
     "@types/chai": "^4.3.16",
     "@types/dockerode": "^3.3.29",
     "@types/fs-extra": "^11.0.4",
-    "@types/node": "^20.13.0",
+    "@types/node": "^20.14.9",
     "@types/sinon": "^17.0.3",
     "@types/tmp": "^0.2.6",
     "c8": "^9.1.0",
     "chai": "^5.1.1",
     "chai-as-promised": "^8.0.0",
-    "mocha": "^10.4.0",
-    "nodemon": "^3.1.2",
+    "mocha": "^10.5.2",
+    "nodemon": "^3.1.4",
     "sinon": "^18.0.0",
-    "tsx": "^4.11.0",
-    "typescript": "^5.4.5",
+    "tsx": "^4.15.8",
+    "typescript": "^5.5.2",
```

Comment:
This means `@import` clauses in JSDoc, yay! (or at least yay until everything is TS...). Also, better detection of types in `.filter` cases, which have bugged me in the past.